python/paddle/nn/layer/loss.py

```diff
@@ -16,14 +16,124 @@
 import paddle.fluid as fluid
 __all__ = [
     #'NCELoss',
-    #    'CrossEntropyLoss',
+    'CrossEntropyLoss',
     #    'MSELoss',
     'L1Loss',
     #    'NLLLoss',
     #   'BCELoss'
 ]
 
 
+class CrossEntropyLoss(fluid.dygraph.Layer):
+    """
+    This operator implements the cross entropy loss function.
```

Comment:
1. This `CrossEntropyLoss`, combile softmax, cross_entropy, and reduce_sum/mean together, need to describe more clearly.
2. Wheteher this class is the same with PyTorch?   Pytorch combines `log_softmax` and `nll_loss`.

Reply:
DONE